### DISCUSSION TITLE
chrome-plugin-update-url-cleaner/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#146] [Giscus]
# chrome-plugin-update-url-cleaner/

这篇文章介绍了一个名为网址净化器的 Chrome 插件。该插件可以将当前标签页的网址转换成干净的网址。作者在 15 年前开发了一款电商插件，用于净化淘宝链接。他发现当时美工和客服对 URL 机制不太了解，导致店铺装修和发送给客户的链接过长。因此，他重新开发了这个插件，并进行了更新迁移至 Manifest V3、更改插件名称、添加设置白名单功能以及更新图标等操作。目前内置白名单默认支持多个常见网站，并且支持国际化（i18n）。最后，作者还提到在发布 2.0.0 版本后学习到 CSS 属性 field-sizing 并进行优化，并发布了 2.0.1 版本。感兴趣的读者可以前往 Chrome Web Store 安装体验该插件。

https://www.eallion.com/chrome-plugin-update-url-cleaner/

**Comment by W4J1e** [comment]
卧槽我就遇到过，问店铺客服有没有优惠的时候，对方发来一个联盟的优惠券，其实就是有返利的。
很多客服也确实复制链接都是一股脑，不过好像某宝狗东这些支持自动识别展示成卡片了，观感多少好了一些。

**Reply by eallion** [reply]
确实移动电商份额超过 PC 后，这些用户体验就改变了。
